index.js

```diff
@@ -66,9 +74,48 @@ function clickHandle (event) {
         
         let element = document.getElementById('message');
         element.innerHTML = `<p>${data.overview}</p>`;
-        
-        console.log(data);
-        
     });
 }
 
+const renderTopMovies = function() {
+
+    let cardsList = document.getElementsByClassName('movieCard');
+
+    fetch(latestRelease)
+    .then(response => response.json())
+    .then((data) => {
+
+        let i = 0;
+        card = Object.values(cardsList);
+        
+        card.forEach(element => {
```

Comment:
```suggestion
        card.forEach((element, index) => {
```
No necesitas la variable `i`, el segundo parametro del callback que recibe `forEach` es el index. Mira la [documentación](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach).